marimo/_plugins/ui/_impl/table.py

```diff
@@ -137,6 +137,8 @@ class SearchTableResponse:
     data: str
     total_rows: Union[int, Literal["too_many"]]
     cell_styles: Optional[CellStyles] = None
+    # Mapping of rowId -> columnName -> hover text (plain string)
+    cell_hover_texts: Optional[dict[RowId, dict[ColumnName, str]]] = None
```

Comment:
maybe: `Optional[dict[RowId, dict[ColumnName, Optional[str]]]]` to allow for empty hover for cells